### PR TITLE
feat: add automated plant watering workshop for educators

### PR DESCRIPTION
**Pull request by HipsterBrown**
TODO:

- [x] update image assets to match workshop setup
- [x] add details for setting up soil / coffee grounds and water in cups in relation to hardware components (include image examples)
- [x] review process code to match expectations of workshop
- [x] update slide deck with relevant content

### CODE REVIEW

File: site/viamguides/src/workshop-plant-watering/workshop-plant-watering.md [hunk 1]
```diff
@@ -0,0 +1,506 @@
...
+   - **Test the workshop project**: Run through the build following the prepared documentation to ensure it works as expected. Simulate common issues and prepare troubleshooting tips for these scenarios. If physical space is limited, consider sharing supplies.
+   - **Bring backups**: In addition to the backup supplies mentioned earlier during planning.
+
+<!-- ------------------------ -->
+
+## Delivery Plan - overview
+
+Duration: 2
+
+Feel free to make a copy of [this sample slide deck](https://docs.google.com/presentation/d/1BUkv3h-49KYPi-HqHn36P38V67ThM9-UVU-muOQZ0GU/edit#slide=id.g314410d34af_0_25), customize it, and make it your own.
```
Comment (loopDelicious) [inline site/viamguides/src/workshop-plant-watering/workshop-plant-watering.md:119]:
```suggestion
Feel free to make a copy of [this sample slide deck](https://docs.google.com/presentation/d/1_mI__LEwgJApM_3Z2yKwaWOjCPyVPndVNHyrybrDt2g/edit?usp=sharing), customize it, and make it your own.
```

File: site/viamguides/src/workshop-plant-watering/workshop-plant-watering.md [hunk 2]
```diff
@@ -0,0 +1,506 @@
...
+   ![raspberry pi hostname username and password](assets/generalSettings.png)
+1. Select the **SERVICES** tab, check **Enable SSH**, and select **Use password authentication**.
+   ![raspberry pi enable SSH](assets/sshSettings.png)
+   > aside negative
+   > Be sure that you remember the `hostname` and `username` you set, as you will need this when you SSH into your Pi.
+1. **Save** your updates, and confirm `YES` to apply OS customization settings. Confirm `YES` to erase data on the microSD card. You may also be prompted by your operating system to enter an administrator password. After granting permissions to the Imager, it will begin writing and then verifying the Linux installation to the microSD card.
+1. Remove the microSD card from your computer when the installation is complete.
+
+<form>
+  <name>WWhat is the purpose of setting a custom `hostname` during the Raspberry Pi OS configuration?</name>
```
Comment (loopDelicious) [inline site/viamguides/src/workshop-plant-watering/workshop-plant-watering.md:215]:
```suggestion
  <name>What is the purpose of setting a custom `hostname` during the Raspberry Pi OS configuration?</name>
```

File: site/viamguides/src/workshop-plant-watering/workshop-plant-watering.md [hunk 3]
```diff
@@ -0,0 +1,506 @@
...
+
+<!-- ------------------------ -->
+
+## Delivery Plan - Hands-on Experiment
+
+Duration: 40
+
+### Test the moisture sensor
+
+1. In [the Viam app](https://app.viam.com/fleet/locations) under the **CONTROL** tab, you can see the hardware component called `board-1``. That's your Raspberry Pi that you configured on the **CONFIGURE** tab.
```
Comment (loopDelicious) [inline site/viamguides/src/workshop-plant-watering/workshop-plant-watering.md:293]:
```suggestion
1. In [the Viam app](https://app.viam.com/fleet/locations) under the **CONTROL** tab, you can see the hardware component called `board-1`. That's your Raspberry Pi that you configured on the **CONFIGURE** tab.
```

File: site/viamguides/src/workshop-plant-watering/workshop-plant-watering.md [hunk 4]
```diff
@@ -0,0 +1,506 @@
...
+
+   > aside negative
+   > The moisture threshold is adjusted by turning the screw on the top of the sensor module. As you get into the Control interface of the Viam app, you can refine the threshold based on the needs of your plant.
+
+   > aside negative
+   > If any problems occur, check under the **LOGS** tab to see what might be going wrong.
+
+### Test the pump
+
+1. In [the Viam app](https://app.viam.com/fleet/locations) under the **CONTROL** tab, you can see the hardware component called `board-1``. That's your Raspberry Pi that you configured on the **CONFIGURE** tab.
```
Comment (loopDelicious) [inline site/viamguides/src/workshop-plant-watering/workshop-plant-watering.md:306]:
```suggestion
1. In [the Viam app](https://app.viam.com/fleet/locations) under the **CONTROL** tab, you can see the hardware component called `board-1`. That's your Raspberry Pi that you configured on the **CONFIGURE** tab.
```

File: site/viamguides/src/workshop-plant-watering/workshop-plant-watering.md [hunk 5]
```diff
@@ -0,0 +1,506 @@
...
+      "env": {
+        "BOARD_NAME": "board-1",
+        "ROBOT_API_KEY": "your-api-key",
+        "ROBOT_API_KEY_ID": "your-api-key-id",
+        "ROBOT_ADDRESS": "your-robot-address"
+      },
+    ```
+    ![configure JSON](assets/configureJSON.png)
+    > aside negative
+    > The `BOARD_NAME` is the default names for the Raspberry Pi when added to our Viam machine. Other machine credentials can be found under the **CONNECT** tab, selecting an SDK, and toggling **Include API key** to reveal your credentials within the code sample.
```
Comment (loopDelicious) [inline site/viamguides/src/workshop-plant-watering/workshop-plant-watering.md:385]:
```suggestion
    > The `BOARD_NAME` is the default name for the Raspberry Pi when added to our Viam machine. Other machine credentials can be found under the **CONNECT** tab, selecting an SDK, and toggling **Include API key** to reveal your credentials within the code sample.
```

File: site/viamguides/src/workshop-plant-watering/workshop-plant-watering.md [hunk 6]
```diff
@@ -0,0 +1,506 @@
...
+   - Why is it important to connect your machine to the Viam app during setup?
+   - What does it mean that `viam-server` enables remote access to the Raspberry Pi?
+   - How does the Viam app simplify the process of testing and controlling hardware components like the board?
+1. **Hands-on Experiment**
+   - What is the primary role of `viam-server` in the system architecture of your plant watering setup?
+   - What is the purpose of the `env` field in your process configuration?
+
+### Next-level projects
+
+Here are some ideas for expanding and enhancing the current project to create autonomous behaviors.
```
Comment (loopDelicious) [inline site/viamguides/src/workshop-plant-watering/workshop-plant-watering.md:487]: Consider adding a link to this project (Nicolai's Viam video): https://youtu.be/l4MQRiHPY6Y?feature=shared

File: site/viamguides/src/plant-watering-device-workshop/plant-watering-device-workshop.md [hunk 7]
```diff
@@ -7,7 +7,7 @@ status: Published
 feedback link: <https://github.com/viam-labs/viamcodelabs/issues>
 tags: Getting Started, Developer, Hardware
 
-# Plant Watering Device Workshop
+# Automatic plant watering
```
Comment (loopDelicious) [inline site/viamguides/src/plant-watering-device-workshop/plant-watering-device-workshop.md:10]: Can I convince you to rename these 2 codelabs to something more along the lines of Plant Watering? With moisture sensors maybe?
I suspect we'll have a number of Automate/Automatic titles.

**Review by loopDelicious — COMMENTED** [review]
Everything looks great!
- Are there any protocols that need to be enabled on the Pi?
- One note on the slide deck (14) - `NO (on relay)` - is that correct? I saw `Normally Open` in the codelab text.

**Comment by HipsterBrown** [comment]
> Are there any protocols that need to be enabled on the Pi?

No, all components communicate over direct GPIO access that's available by default. 

> One note on the slide deck (14) - NO (on relay) - is that correct? I saw Normally Open in the codelab text.

In the slide, I wanted to be clear in the slide that the NO connection is on the relay module, which is more spelled out in the codelab text.

**Pull request merged**